static/_redirects

```diff
@@ -109,23 +109,22 @@
 /guides/artifacts/artifacts-core-concepts /guides/artifacts 301
 /guides/artifacts/artifacts-faqs /support/index_artifacts 301
 /guides/artifacts/dataset-versioning /guides/artifacts 301
-/guides/artifacts/model-versioning /guides/model_registry 301
+/guides/artifacts/model-versioning /guides/core/registry/model_registry 301
```

Comment:
Not working, goes to https://docs-1245.docodile.pages.dev/guides/models/tables/registry/model_registry